stdlibs/std/addr_set.gno

```diff
@@ -13,35 +17,25 @@ type AddressSet interface {
 
 //----------------------------------------
 // AddressList implements AddressSet.
-// TODO implement AddressTree with avl.
```

Comment:
Agree, but then why does `AddressList` have a condition that avoids double inclusions?

Reply:
Ahah, good question, `List` should support duplicates; but since it implements the `Set`, it may make sense to just considering as a set too, and then, the question is: how important/useful is it to keep the `List` implementation and not only the `Tree` one?

/summon @jaekwon

Reply:
Ok perfect, i'll work on it !

Reply:
@moul updated, keep me in touch if you wish some updates

ps: commits are rebased and fixed-up